message_definitions/v1.0/messages.xml

```diff
@@ -1747,17 +1751,19 @@
     </message>
 
     <message name="IMU_GYRO_RAW" id="203">
-      <field name="id" type="uint8"/>
-      <field name="gp" type="int32" unit="adc"/>
-      <field name="gq" type="int32" unit="adc"/>
-      <field name="gr" type="int32" unit="adc"/>
+      <field name="id"    type="uint8"/>
+      <field name="temp"  type="float" unit="Celcius"/>
+      <field name="gp"    type="int32" unit="adc"/>
+      <field name="gq"    type="int32" unit="adc"/>
+      <field name="gr"    type="int32" unit="adc"/>
     </message>
 
     <message name="IMU_ACCEL_RAW" id="204">
-      <field name="id" type="uint8"/>
-      <field name="ax" type="int32" unit="adc"/>
-      <field name="ay" type="int32" unit="adc"/>
-      <field name="az" type="int32" unit="adc"/>
+      <field name="id"    type="uint8"/>
+      <field name="temp"  type="float" unit="Celcius"/>
```

Comment:
Is there a particular reason to add `temp` before the sensor values ? If someone was doing "manual" parsing of the data, it will shift the values

Reply:
No reason, just because it looked nicer to place it next to the id. Since I changed to add the id anyways with the last multi-imu pull request, does it really matter for this one? I don't think many things paarse the RAW messages anyways except our calibration tools, which should work fine.

For the Matlab and Python scripts we use it should not matter, because they use the names of the fields. I think we should assume people do this for best practices.